apko-publish/action.yaml

```diff
@@ -135,10 +135,12 @@ runs:
         --network host \
         -v $PWD:${{ github.workspace }} \
         -v /tmp:/tmp \
+        -v $GITHUB_OUTPUT:$GITHUB_OUTPUT \
         --workdir ${{ github.workspace }} \
         -e "GITHUB_ACTOR=${{ inputs.repository_owner }}" \
         -e "GITHUB_TOKEN=${{ inputs.token }}" \
         -e "REPOSITORY=${{ inputs.repository }}" \
+        -e "GITHUB_OUTPUT=$GITHUB_OUTPUT" \
```

Comment:
and this?

Reply:
Not strictly required, both are required if digest output (line 180) is done inside the docker run command. An alternative way could be something like this https://github.com/andros21/actions/pull/4/commits/0024871c27dd41f5ba3ee4d3cc83d3a2c9ec5718. If ok, I can update it

Reply:
i think you can remove this